content/news/XXXX-XX-XX-mixxx-2-3-0-released.md

```diff
@@ -0,0 +1,138 @@
+
+Prepare your DJ set using [hotcue colors & labels]({filename}/news/2020-08-25-new-in-2-3-hotcue-colors.md), mark [intro/outro sections]({filename}/news/2020-07-09-intro-outro-sections.md) in your tracks and add [track colors]({filename}/news/2020-10-24-new-in-2-3-track-colors.md) for easier library navigation.
+Our new multithreaded analysis and [more accurate key detection]({filename}/news/2021-04-08-new-in-2-3-keyfinder.md) will improve the preparation process further.
+
+Mixxx 2.3.0 comes with a new default skin:
+"LateNight" underwent a massive redesign and replaces "Deere" as default skin, so check out the [screenshots page]({filename}/pages/screenshots.md).
+
+If you're a Rekordbox or Serato user, switching to Mixxx becomes a lot easier:
+You can now play tracks directly from USB drives that contain [Rekordbox]({filename}/news/2020-07-20-new-in-2-3-rekordbox-support.md) and [Serato](2021-02-08-new-in-2-3-serato-support.md) libraries.
+
+We also added support for recording/streaming in the [Opus codec](https://opus-codec.org/), introduced [deck cloning]({filename}/news/2020-11-15-new-in-2-3-deck-clone.md), polished the library and preferences (including the controller workflow) and [switched to the CMake build system generator]({filename}/news/2020-12-14-scons-cmake-migration.md).
+
+2.3.0 also adds out-of-the-box support for the [Pioneer DDJ-200](https://manual.mixxx.org/2.3/en/hardware/controllers/pioneer_ddj_200.html) and [DDJ-400](https://manual.mixxx.org/2.3/en/hardware/controllers/pioneer_ddj_400.html), the [Native Instruments Traktor Kontrol S3](https://manual.mixxx.org/2.3/en/hardware/controllers/native_instruments_traktor_kontrol_s3.html), the [Hercules DJControl Inpulse 200](https://manual.mixxx.org/2.3/en/hardware/controllers/hercules_djcontrol_inpulse_200.html) and [Jogvision](https://manual.mixxx.org/2.3/en/hardware/controllers/hercules_djcontrol_jogvision.html), the [Roland DJ-505](https://manual.mixxx.org/2.3/en/hardware/controllers/roland_dj_505.html), the [Behringer B-Control BCR2000](https://manual.mixxx.org/2.3/en/hardware/controllers/behringer_bcr2000.html) and [DDM4000](https://manual.mixxx.org/2.3/en/hardware/controllers/behringer_ddm4000.html), the [ION Discover DJ Pro](https://manual.mixxx.org/2.3/en/hardware/controllers/ion_discover_dj_pro.html) and the [Numark iDJ Live II](https://manual.mixxx.org/2.3/en/hardware/controllers/numark_idj_live_ii.html). A few existing controller mappings have received fixes and new features, too.
+
+For the full list of changes, have a look at the [changelog](https://manual.mixxx.org/2.3/en/chapters/appendix/version_history.html) and the [2.3.0 milestone on Launchpad](https://launchpad.net/mixxx/+milestone/2.3.0).
+
+This has been a rather big release, with lots of useful changes, bug fixes and improvements.
+All in all, a total of 7183 changes changed over 1 million lines of code were changed since the [2.2.4 release]({filename}/news/2020-06-11-Mixxx-2-2-4-released.md)!
```

Comment:
```suggestion
All in all, with a total of 7183 changes over 1 million lines of code were modified since the [2.2.4 release]({filename}/news/2020-06-11-Mixxx-2-2-4-released.md)!
```